cli/src/commands/util.rs

```diff
@@ -35,6 +36,7 @@ pub(crate) enum UtilCommand {
     Mangen(UtilMangenArgs),
     MarkdownHelp(UtilMarkdownHelp),
     ConfigSchema(UtilConfigSchemaArgs),
+    Exec(UtilExecArgs),
```

Comment:
nit: the list should (have been) sorted. I noticed util.rs isn't split up to per-command files yet, I'll send a patch separately.